draft-ietf-quic-load-balancers.md

```diff
@@ -317,6 +317,11 @@ There are situations where a server pool might be operating two or more routing
 algorithms or parameter sets simultaneously.  The load balancer uses the first
 two bits of the connection ID to multiplex incoming DCIDs over these schemes.
 
+Encryption in the algorithms below uses the AES-128-ECB cipher. Future standards
+may deprecate these algorithms and replace them with others that use other
+ciphers, in accordance with {{?RFC7696}}. QUIC-LB implementations SHOULD be
```

Comment:
```suggestion
Encryption in the algorithms below uses the AES-128-ECB cipher. Future standards
could add new algorithms that use other ciphers to provide cryptographic agility in
accordance with {{?RFC7696}}. QUIC-LB implementations SHOULD be
```
No need to mention deprecation; the important factor here is that you have a path to introducing new algorithms.